examples/asr/asr_vad/feature_rttm_to_text.py

```diff
@@ -0,0 +1,381 @@
+        )
+    else:
+        logging.info("Using raw audios as input...")
+        config = {
+            "manifest_filepath": cfg.dataset_manifest,
+            "batch_size": cfg['batch_size'],
+            "num_workers": cfg.get('num_workers', 0),
+        }
+        dataloader = asr_model._setup_transcribe_dataloader(config)
+
+    logging.info(f"Transcribing...")
+
+    # setup AMP (optional)
+    if cfg.amp and torch.cuda.is_available() and hasattr(torch.cuda, 'amp') and hasattr(torch.cuda.amp, 'autocast'):
+        logging.info("AMP enabled!\n")
+        autocast = torch.cuda.amp.autocast
+    else:
+
+        @contextlib.contextmanager
+        def autocast():
+            yield
+
+    hypotheses = []
+    all_hypotheses = []
+    with profile_fn(
+        activities=[ProfilerActivity.CPU, ProfilerActivity.CUDA], record_shapes=True, profile_memory=True
+    ) as prof:
+        t0 = time.time()
+        with autocast():
+            with torch.no_grad():
+                with record_fn("infer_loop"):
+                    for test_batch in tqdm(dataloader, desc="Transcribing"):
+                        with record_fn("infer_model"):
+                            if cfg.use_feature:
+                                outputs = asr_model.forward(
+                                    processed_signal=test_batch[0].to(map_location),
+                                    processed_signal_length=test_batch[1].to(map_location),
+                                )
+                            else:
+                                outputs = asr_model.forward(
+                                    input_signal=test_batch[0].to(map_location),
+                                    input_signal_length=test_batch[1].to(map_location),
+                                )
+
+                        with record_fn("infer_other"):
+                            logits, logits_len = outputs[0], outputs[1]
+
+                            current_hypotheses, all_hyp = decode_function(logits, logits_len, return_hypotheses=False,)
+
+                            hypotheses += current_hypotheses
+                            if all_hyp is not None:
+                                all_hypotheses += all_hyp
+                            else:
+                                all_hypotheses += current_hypotheses
+
+                            del logits
+                            del test_batch
+        t1 = time.time()
+    logging.info(f"Time elapsed: {t1 - t0: .2f} seconds")
+    if cfg.profiling:
+        print(prof.key_averages().table(sort_by="cuda_time_total", row_limit=10))
+        print("--------------------------------------------------------------------\n")
+
+    # Save output to manifest
+    manifest_data = load_manifest(cfg.dataset_manifest)
+    groundtruth = []
+    for i in range(len(manifest_data)):
+        manifest_data[i]["pred_text"] = hypotheses[i]
+        groundtruth.append(clean_label(manifest_data[i]["text"]))
```

Comment:
We should remove implicit normalization here, or have it default off and turn on with an input argument. Can cause confusion in future when there is a mismatch in WER.